src/coffee/run.coffee

```diff
@@ -244,7 +244,10 @@ module.exports = class
               importer.continueOnProblems = opts.continueOnProblems
               importer.changeState(publish, remove, filterFunction)
             .then (result) ->
-              _consoleWarnAllResults result
+              if result.length > 500
```

Comment:
If the products are more than 500, it should write to a file.

Reply:
Interesting, does it crash at 500 logging outputs already?

Reply:
No, it doesn't.

Reply:
okay, so what is the reason for putting the limit at 500 already? Does seem a bit low to me.

Reply:
It will only output 100.